autotest/tools/pipeline/test_pipeline_chat_turbomind_vl.py

```diff
@@ -32,3 +32,20 @@ def test_pipeline_chat_tp2(config, model, worker_id):
     p.start()
     p.join()
     assert_pipeline_vl_chat_log(config, model)
+
+
+@pytest.mark.pipeline_chat
+@pytest.mark.gpu_num_1
+@pytest.mark.pr_test
+@pytest.mark.parametrize('model', [
+    'liuhaotian/llava-v1.6-vicuna-7b', 'OpenGVLab/InternVL2-4B',
+    'OpenGVLab/InternVL2-8B', 'internlm/internlm-xcomposer2d5-7b'
+])
+def test_pipeline_pr_test(config, model, worker_id):
```

Comment:
this case is not triggered. something wrong?
```
autotest/tools/quantization/test_quantization_w4a16.py .                 [  6%]
autotest/tools/convert/test_convert.py ..                                [ [20](https://github.com/InternLM/lmdeploy/actions/runs/10244807978/job/28338525428?pr=2192#step:9:21)%]
autotest/tools/pipeline/test_pipeline_chat_pytorch.py .                  [ 26%]
autotest/tools/pipeline/test_pipeline_chat_turbomind.py ..               [ 40%]
autotest/tools/restful/test_restful_chat_hf_turbomind.py ..              [ 53%]
autotest/tools/restful/test_restful_chat_workspace.py ..                 [ 66%]
autotest/tools/chat/test_command_chat_hf_pytorch.py .                    [ 73%]
autotest/tools/chat/test_command_chat_hf_turbomind.py ..                 [ 86%]
autotest/tools/chat/test_command_chat_workspace.py ..                    [100%]
```

Reply:
> this case is not triggered. something wrong?
> 
> ```
> autotest/tools/quantization/test_quantization_w4a16.py .                 [  6%]
> autotest/tools/convert/test_convert.py ..                                [ [20](https://github.com/InternLM/lmdeploy/actions/runs/10244807978/job/28338525428?pr=2192#step:9:21)%]
> autotest/tools/pipeline/test_pipeline_chat_pytorch.py .                  [ 26%]
> autotest/tools/pipeline/test_pipeline_chat_turbomind.py ..               [ 40%]
> autotest/tools/restful/test_restful_chat_hf_turbomind.py ..              [ 53%]
> autotest/tools/restful/test_restful_chat_workspace.py ..                 [ 66%]
> autotest/tools/chat/test_command_chat_hf_pytorch.py .                    [ 73%]
> autotest/tools/chat/test_command_chat_hf_turbomind.py ..                 [ 86%]
> autotest/tools/chat/test_command_chat_workspace.py ..                    [100%]
> ```

![image](https://github.com/user-attachments/assets/7c0a4b3c-3f90-4306-9aba-e87f983b138f)
here. vl testcase use 1 GPU, In order to maximize the use of GPU, the use case is executed in two steps.

Reply:
why is it not shown as the first one?

Reply:
> why is it not shown as the first one?

-n 2 which means run in parallel is used. By using -n test process show like this.